demo/index.html

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
```

Comment:
Yes webpack-dev-server needs a short index.html in the root directory to run the bench tests.

Since examples are broken on master and we are moving to a demo structure anyway, I moved this index file into the intended target directory now.